From 23de53b42a52c2847d74b104c8b1d2208f30181b Mon Sep 17 00:00:00 2001
From: Linu Cherian <lcherian@marvell.com>
Date: Thu, 2 Apr 2020 17:11:34 +0530
Subject: [PATCH 5/6] octeontx2-af: Add new silicon variant Loki

commit 095eb61dff783b1df460f6837ca5d4f38b4afbcc from
git@git.assembla.com:cavium/WindRiver.linux.git

Add Loki to the list of silicons that are post 96xx C0.

Change-Id: I893b48b51c9d22d075fb0692562fc8d9688ca5c8
Signed-off-by: Linu Cherian <lcherian@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/26232
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/rvu.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/rvu.h b/drivers/net/ethernet/marvell/octeontx2/af/rvu.h
index a318f6075b70..a66da9254029 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/rvu.h
+++ b/drivers/net/ethernet/marvell/octeontx2/af/rvu.h
@@ -508,7 +508,8 @@ static inline bool is_rvu_post_96xx_C0(struct rvu *rvu)
 {
 	struct pci_dev *pdev = rvu->pdev;
 
-	return (pdev->revision == 0x08) || (pdev->revision == 0x30);
+	return (pdev->revision == 0x08) || (pdev->revision == 0x30) ||
+		(pdev->revision == 0x20);
 }
 
 static inline bool is_rvu_96xx_A0(struct rvu *rvu)
-- 
2.17.1

